lucien-ios/Dashboard/LendingCollectionViewCell.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LendingCollectionViewCell.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/10/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+class LendingCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var lendedComicTitle: UILabel!
```

Comment:
should be private

Reply:
Should be suffixed with `label` e.g. `lendedComicTitleLabel`. Can also probably shorten to `titleLabel` since there aren't any other titles in the cell